reco_utils/docker/pyspark/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+ARG BASE_CONTAINER=jupyter/pyspark-notebook
```

Comment:
are you intending to allow these ARGs to be changed at build time?

Reply:
Now the base image is fixed. Previously I was thinking having different base for different images.